frontend/webapp/core/__init__.py

```diff
@@ -68,16 +68,16 @@ def setup():
 
     modules = Modules_state()
 
-    try:
-        app.artemis_logger.debug('Starting Database..')
-        modules.call('database', 'start')
-
-        if not modules.is_up_or_running('database'):
-            app.artemis_logger.error('Couldn\'t start Database.')
-            exit(-1)
-    except BaseException:
-        app.artemis_logger.exception('exception while starting Database')
-        exit(-1)
+    # try:
```

Comment:
What are the effects of commenting this code?

Reply:
it doesn't work for multiple database modules. need to follow the approach of detection.

@pgigis we need to check if group is running and not just one instance. Also, when a user starts multiple detectors they start one-by-one. Can we do it concurrently?